contracts/src/contracts/nullifier_registry/nullifier_registry.cairo

```diff
@@ -1 +1,294 @@
+#[starknet::contract]
+pub mod NullifierRegistry {
+    use starknet::{
+        ContractAddress, get_caller_address, get_block_timestamp,
+        storage::{Map, StorageMapReadAccess, StorageMapWriteAccess}
+    };
+    use zkramp::contracts::nullifier_registry::interface::{
+        INullifierRegistry, INullifierRegistryDispatcher, INullifierRegistryDispatcherTrait
+    };
+
+    #[storage]
+    pub struct Storage {
+        // / map(index -> writers address)
+        writers: Map<u64, ContractAddress>,
```

Comment:
Ig you can use [`Vec`](https://github.com/starkware-libs/cairo/blob/main/corelib/src/starknet/storage/vec.cairo) here

Reply:
@m-kus can give me an example here, 

i try using this 

` writers: Vec<ContractAddress>,`

and perform appending the address 

`self.all_writers.append(new_writer);`

am getting the following errors

```
Method `append` could not be called on type `core::starknet::storage::storage_base::StorageBase::<core::starknet::storage::Mutable::<core::starknet::storage::vec::Vec::<core::starknet::contract_address::ContractAddress>>>`.
Candidate `ArrayTrait::append` inference failed with: Type mismatch: `core::starknet::storage::storage_base::StorageBase::<core::starknet::storage::Mutable::<core::starknet::storage::vec::Vec::<core::starknet::contract_address::ContractAddress>>>` and `core::array::Array::<?30>`.
Candidate `ByteArrayTrait::append` inference failed with: Type mismatch: `core::starknet::storage::storage_base::StorageBase::<core::starknet::storage::Mutable::<core::starknet::storage::vec::Vec::<core::starknet::contract_address::ContractAddress>>>` and `core::byte_array::ByteArray`.
```

Reply:
Probably need to import `MutableVecTrait`